src/richie/apps/search/views.py

```diff
@@ -24,3 +27,20 @@ def bootstrap_elasticsearch(request, version):
 
     messages.info(request, _("The search index was successfully bootstrapped"))
     return Response({})
+
+
+@api_view(["GET"])
+@cache_page(60 * 60 * 2)
```

Comment:
maybe add a test that mock `get_static_definitions` and ensures that it is not called on the second request to this url?

Reply:
Done 👍